src/apis/apis.ts

```diff
@@ -8,31 +8,38 @@ import type {
   TradeStatusCodes
 } from '@types'
 
-export const getMyProfile = async (): Promise<AxiosResponse<MyProfile>> => {
-  const res = await axios.get('/members/mypage')
-  return res.data
+export const getMyProfile = async () => {
+  const res = await http.get<null, MyProfile>('/members/mypage')
+  return res
 }
```

Comment:
api 선언부일뿐이라 `async/await` 제거해두 될 것 같습니다 !
추가로 즉시 return하는것은 어떠신가요  ? 

```suggestion
export const getMyProfile = () => http.get<null, MyProfile>('/members/mypage')
```

Reply:
[완료](https://github.com/price-offer/offer-fe/pull/168/commits/052535f0060c37f6a55d7dd9ea59a2586249914a#diff-36514db8adf960d228ccdef4ecc7c2265c05784edd10bdf859260049829c092c)